slave.mk

```diff
@@ -109,6 +110,7 @@ ifneq ($(CONFIGURED_PLATFORM),generic)
 endif
 
 configure :
+	$(Q)mkdir -p $(CONTAINER_TMP_PATH)
```

Comment:
Looks like it is not necessary to create the dir /tmp, the /tmp path should always exist.

Reply:
Thanks, removed this lne.